java-generator/it/src/it/cert-manager/verify.groovy

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import static org.junit.jupiter.api.Assertions.assertEquals
+
+// TODO: This should be modified to test against the current extension sources
+assertEquals(
```

Comment:
should we use `approvaltests` here?

Reply:
I tried to, but it didn't work (Approval tests relies on a JUnit execution environment).

Anyway, considering the way we are using Approvaltests, this is practically the same (minus the dependency)

Reply:
There is a port for Groovy:
https://github.com/rzabini/ApprovalTests.Spock

What I'm enjoining from the Approval tests is the workflow with `approved` and `received` files ... but now fussy on it, especially at this stage.